Fix for Travis-CI OSX builds due to RVM using "cd"

kedarbhat opened the pull request.

jeking3 commented:
It would be nice to enable osx builds again, I couldn't figure out why "cd" was broken there.  Is this all that was needed?

kedarbhat commented:
Yea. pushd/popd are also overrwritten by rvm, but something else broke later in the process when I added them to the unset arguments, though I'm having trouble finding the Travis build which would show what exactly broke. In any case, I removed pushd/popd from the unset command and everything seemed to work (since "cd" was the main issue).
